model_analyzer/config/generate/model_variant_name_manager.py

```diff
@@ -41,6 +41,17 @@ def from_dict(
 
         return model_variant_name_manager
 
+    @staticmethod
+    def make_ensemble_submodel_key(
+            ensemble_config_dicts: List[Dict]) -> Dict[str, str]:
+        ensemble_names = [
+            ensemble_config_dict["name"]
+            for ensemble_config_dict in ensemble_config_dicts
+        ]
+        ensemble_key = ','.join(ensemble_names)
+
+        return {"key": ensemble_key}
```

Comment:
Why is the key a dictionary instead of just a string?

Reply:
I'm still confused by this.

Reply:
For type compatibility when writing into the MVNM

Reply:
ok, I see now where a dict is used.